assets/css/_custom.scss

```
@@ -1,11 +1,23 @@
-.figure-cell {
-    display: table-cell !important;
+
+.page {
+    max-width: 100%;
+}
+
+.header-title .tw-mr-1 img {
+    transform: translateY(5px);
+}
+.header-title span.ti-cursor {
+    transform: translateY(-2.5px);
 }
 
 body {
     &[theme=dark] {
-        .invert {
-            filter: invert(83.14%);
+        .invert img {
+            filter: invert(93.16%);
         }
     }
+}
+
+img {
+    display: unset;
 }
```

```suggestion
}

.dark .single .post-meta a {
    color: var(--single-link-color) !important;
}
```

Ajoute la couleur bleu en thème sombre pour les dossiers et auteurs.